src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx

```diff
@@ -119,6 +119,11 @@ export default class ArcanaTracking extends Module {
 	}
 
 	normalise(events: Event[]) {
+		// The whole point of this module was to support "pre-5.3" Three-card Sleeve Draw
+		if (this.parser.patch.before('5.3')) {
+			return events
+		}
+
```

Comment:
If this only was needed before 5.3, should the conditional not be `!this.parser.patch.before('5.3')`? This is ignoring it for before 5.3 instead of including and after.